tests/imap_tests.in

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export EMAILS=""
+export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
+export FACTS_SOURCE="$TEST_DIR/img_test.lf"
```

Comment:
Shouldn't this be 

```bash
export FACTS_SOURCE="$TEST_DIR/imap_tests.lf
                               ^^^
```

instead of 

```bash
export FACTS_SOURCE="$TEST_DIR/img_test.lf"
                               ^^^
```

Not sure if you accidentally committed ``imap_tests.lf``, or if you intended on using it in this test.

Also imap is a little misleading since IMAP usually refers to the mail protocol. Maybe rename these test files to ``img_map_test`` or something?